src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj

```diff
@@ -69,11 +66,6 @@
              Link="Microsoft\Win32\SafeHandles\SafeRegistryHandle.Windows.cs" />
     <Compile Include="System\Security\AccessControl\RegistrySecurity.Windows.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetPlatformIdentifier)' != 'windows'">
```

Comment:
Can you open an issue about recombining the Windows-specific files back into the main file, e.g. we no longer need the separation between RegistryKey.cs and RegistryKey.Windows.cs... the latter can be folded into the former, the "Core" methods consolidated back into their callers, etc.

Reply:
#77974